corehq/blobs/migrate.py

```diff
@@ -233,13 +234,17 @@ class BlobDbBackendMigrator(BaseDocMigrator):
 
     def __init__(self, slug, couchdb, filename=None):
         super(BlobDbBackendMigrator, self).__init__(slug, couchdb, filename)
-        self.db = get_blob_db()
         self.total_blobs = 0
         self.not_found = 0
         if not isinstance(self.db, MigratingBlobDB):
             raise MigrationError(
                 "Expected to find migrating blob db backend (got %r)" % self.db)
 
+    @property
+    @memoized
+    def db(self):
+        return get_blob_db()
```

Comment:
That was because I was inheriting the exporter from the migrator which would check the db against `MigratingBlobDB`. changed in  55f06ad & b8ff4f4